mysql-test/mysql-test-run.pl

```diff
@@ -5984,9 +5989,9 @@ ($)
   vardir=DIR            The directory where files generated from the test run
                         is stored (default: ./var). Specifying a ramdisk or
                         tmpfs will speed up tests.
-  mem                   Run testsuite in "memory" using tmpfs or ramdisk
-                        Attempts to find a suitable location
-                        using a builtin list of standard locations
+  mem[=DIR]             Run testsuite in "memory" using tmpfs or ramdisk
+                        Attempts to use DIR first if specified else
+                        uses as builtin list of standard locations
```

Comment:
@grooverdan , if it's a typo and you want to fix it now, please while you're at it rebase the patch to 10.0, since the JIRA entry says that it should be fixed in 10.0. If you don't want to fix the typo now (not a big deal, it can be done later), no need to bother rebasing, I already have the patch in my 10.0 staging tree.